apps/examples/vite.config.ts

```diff
@@ -82,9 +83,14 @@ function parseFrontMatter(data: unknown, fileName: string) {
 		throw new Error(`Frontmatter key 'component' must be string in ${fileName}`)
 	}
 
-	const order = 'order' in data ? data.order : null
-	if (order !== null && typeof order !== 'number') {
-		throw new Error(`Frontmatter key 'order' must be number in ${fileName}`)
+	const priority = 'priority' in data ? data.priority : null
+	if (priority !== null && typeof priority !== 'number') {
```

Comment:
minor: we should get rid of `priority !== null`
and prbly same goes for category check below too